include/qdmi/client.h

```diff
@@ -981,6 +1055,329 @@ void QDMI_job_free(QDMI_Job job);
 
 /** @} */ // end of client_job_interface
 
+/** @defgroup client_telemetrysensor_query_interface \
+ *  QDMI Client Telemetry Sensor Query Interface
+ *  @brief Provides functions to query telemetry sensors.
+ *  @details An telemetry sensor query is a task submitted by a client to a
+ * device for querying telemetry sensors, i.e. temperature or power.
```

Comment:
My intuition would suggest the following:
```suggestion
 *  @brief Provides functions to query data from telemetry sensors.
 *  @details An telemetry sensor query is a task submitted by a client to a
 * device for querying data from telemetry sensors, i.e. temperature or power.
```